jupyter_book/commands/__init__.py

```diff
@@ -41,7 +41,7 @@ def main():
 
 
 @main.command()
-@click.argument("path-book")
+@click.argument("path-src")
```

Comment:
can we make the user-facing thing a full word? (`path-source`) we can rename to a shorter name in the code (`path_src`) if we want

Reply:
Ooh. my bad. Will do that. Thanks